data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container

```diff
@@ -14,7 +14,7 @@ Volume=/opt/openshift:/opt/openshift:z,ro
 Volume=/opt/openshift/tls/ironic/:/certs/ironic/:z
 Environment="XDG_RUNTIME_DIR=/run/user/${UID}"
 Environment="KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback"
-Environment="DEPLOY_KERNEL_URL=file:///shared/html/images/ironic-python-agent.kernel"
+Environment="DEPLOY_KERNEL_URL=file:///var/lib/ironic/pxe/vmlinuz"
```

Comment:
Not super critical, but nowadays this variable should be passed to Ironic, not BMO (although I haven't checked if the downstream fork already has the required change).

Reply:
It's not being passed to ironic now, so I guess that change hasn't made it to the bootstrap yet.

Reply:
It's either Ironic or BMO, both work. But the future-proof way is to pass it to Ironic. No urgency, we'll clean it up one day.